<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppPages</class>
 <widget class="QStackedWidget" name="AppPages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <widget class="QWidget" name="page_1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>300</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QPushButton" name="btn_dir_vid">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: rgb(73, 73, 109);
	padding: 8px;
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(76, 130, 255);
}

QPushButton:pressed {
	background-color: rgb(41, 83, 124);
}</string>
           </property>
           <property name="text">
            <string>Abrir</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="line_dir_model">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background-color: rgb(68, 71, 90);
	padding: 8px;
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Diretório do modelo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_dir_model">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: rgb(73, 73, 109);
	padding: 8px;
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(76, 130, 255);
}

QPushButton:pressed {
	background-color: rgb(41, 83, 124);
}</string>
           </property>
           <property name="text">
            <string>Abrir</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="line_dir_video">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background-color: rgb(68, 71, 90);
	padding: 8px;
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
           </property>
           <property name="placeholderText">
            <string>Diretório do vídeo</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="btn_start">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: rgb(73, 73, 109);
	padding: 8px;
	border: 2px solid #c3ccdf;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(76, 130, 255);
}

QPushButton:pressed {
	background-color: rgb(41, 83, 124);
}</string>
           </property>
           <property name="text">
            <string>Iniciar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Treinamento</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_3">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Configuração</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
